cinema/views.py

```diff
@@ -0,0 +1,70 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializers
+
+
+# Create your views here.
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializers(movies, many=True)
+
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializers(data=request.data)
+
+        if serializer.is_valid(raise_exception=True):
```

Comment:
If statement is redundant
```suggestion
        serializer.is_valid(raise_exception=True):
```